import java.util.*;
class exp{

    public static void main(String args[]){
        
        try{
            int num1,num2,res;
            num1=23;
            num2=0;
            res=num1/num2;
            System.out.println("result="+res);
        }catch(Exception e){
            System.out.println("exception occur");
        }
        System.out.println("work is done!");
        }
        }
